HW_1/task_4.py

```diff
@@ -0,0 +1,17 @@
+# Задача 4. Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, 
+#если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+#*Пример:*
+# 6 -> 1  4  1
+#24 -> 4  16  4
+# 60 -> 10  40  10
+
+s=int(input("Введите количество журавликов, которое сделали все дети: "))
+if s >= 6 and s % 6 == 0:
+     print('Количество журавликов, которые сделал Петя  ', s // 6)
+     print('Количество журавликов, которые сделала Катя ', s // 6 * 4)
+     print('Количество журавликов, которые сделал Сережа ', s // 6)
```

Comment:
У вас трижды вычисляется одно и то же выражение. И на вычисление каждого будут выделяться ресурсы, чтобы такого не происходило, используют переменные.
```suggestion
    var = s // 6
     print('Количество журавликов, которые сделал Петя  ', var)
     print('Количество журавликов, которые сделала Катя ', var * 4)
     print('Количество журавликов, которые сделал Сережа ', var)
```